assets/src/models/detour.ts

```diff
@@ -1,6 +1,65 @@
 import { LatLngLiteral } from "leaflet"
 import { ShapePoint, Stop } from "../schedule"
 import { CreateDetourMachineInput } from "./createDetourMachine"
+import { array, Infer, nullable, number, string, type } from "superstruct"
+
+export const SimpleDetour = type({
+  route: string(),
+  direction: string(),
+  name: string(),
+  intersection: string(),
+  updatedAt: number(),
+})
+
+export type SimpleDetour = Infer<typeof SimpleDetour>
+
+export const SimpleDetourFromApi = type({
+  route: string(),
+  direction: string(),
+  name: string(),
+  intersection: string(),
+  updated_at: number(),
+})
+
+export type SimpleDetourFromApi = Infer<typeof SimpleDetourFromApi>
+
+const simpleDetourFromData = (
+  detourData: SimpleDetourFromApi
+): SimpleDetour => ({
+  route: detourData.route,
+  direction: detourData.direction,
+  name: detourData.name,
+  intersection: detourData.name,
```

Comment:
```suggestion
  intersection: detourData.intersection,
```

Reply:
Fixes [this](https://github.com/mbta/skate/pull/2757#issuecomment-2332166168), I think.

Reply:
Ahhhh! I think this was a regression in my model update, thanks for catching